docs/speech.rst

```diff
@@ -46,7 +46,6 @@ Functions
     emphasis.
 
 .. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128)
-                 pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128, pin=(pin_speaker, pin0))
```

Comment:
The speech module will still have the pin argument in V2, it's just the default that changes.